src/main/java/io/confluent/connect/hdfs/parquet/ParquetHiveUtil.java

```diff
@@ -79,8 +88,17 @@ private Table constructParquetTable(
     }
     // convert Connect schema schema to Hive columns
     List<FieldSchema> columns = HiveSchemaConverter.convertSchema(schema);
+    List<FieldSchema> partitions = partitioner.partitionFields();
+    for (FieldSchema partition : partitions) {
+      for (FieldSchema column : columns) {
+        if (partition.getName().equals(column.getName())) {
+          columns.remove(column);
+          break;
+        }
+      }
+    }
```

Comment:
Adding a helper method for this would be great so that we can avoid copying the code on the two instances.